Add key and cert location to example TLS config (#3916)
influxdata/docs-v2: content/enterprise_influxdb/v1.9/administration/configure/security/enable_tls.md

The best practice in terms of security is to transfer the certificate to the client and make it trusted
(either by putting in the operating system's trusted certificate system or using the `ssl_ca` option).
The alternative is to sign the certificate using an internal CA and then trust the CA certificate.

> Then add the key and certificate to the InfluxDB output plugin configuration file as shown below.

Reviewer: ```suggestion
Provide the file paths of your key and certificate to the InfluxDB output plugin as shown below.
```
